api/models/call_log_event.py

```diff
@@ -4,11 +4,8 @@
 
 
 class CallLogEventQuery(BaseQuery):
-    def call_sid_exist(self, form_data):
-        return (
-            self.filter(CallLogEvent.call_sid == form_data["CallSid"]).first()
-            is not None
-        )
+    def call_sid_exist(self, call_sid):
+        return self.filter(CallLogEvent.call_sid == call_sid).first() is not None
```

Comment:
Updated the query based on the changes done in the handle event service from where the query is called.